user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -1216,15 +1240,97 @@ define([
         obj: $target,
         time: (new Date()).getTime()
       });
+      $(".folder.selected").children(".element").attr("tabindex",-1).attr("aria-selected",false);
       $(".folder.selected").removeClass("selected");
       $(".folder.secondarySelected").removeClass("secondarySelected");
       $target.addClass("selected");
+      $target.children(".element").attr("tabindex",0).attr("aria-selected",true);
       //deselect any files
       $(".file.selected").removeClass("selected");
       depth = $target.attr("path").split("/").length;
       event.stopPropagation();
     },
 
+    keyDownFolder: function (event) {
+      let keyCode = event.which || event.keyCode;
+      if (keyCode === 13 || keyCode === 32){
```

Comment:
You should use constants for these key codes (declared at the top of the AMD module).

Reply:
Example: https://github.com/pentaho/pentaho-analyzer/blob/bb48e405d0d25fa3a5b3876abbbdf585bbca9a2e/client/src/main/javascript/scripts/report/cv_rptConst.js#L63-L79

Reply:
Another good use for the proposed a11y AMD module.

Reply:
done. will be creating a PR.